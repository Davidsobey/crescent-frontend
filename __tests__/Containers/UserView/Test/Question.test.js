import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'; // Smart components

// Component to be tested
import UserTestQuestion from '../../../../src/Containers/UserView/Test/index.js';


const mockStore = configureStore();
const initialState = {
  TestReducer:{
    test: {},
  },
  LoginReducer:{
    user: { },
  },
  CourseReducer:{
    course: {},
  },
};
const store = mockStore(initialState);

describe('<UserTestQuestion />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<UserTestQuestion store={store} />);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});