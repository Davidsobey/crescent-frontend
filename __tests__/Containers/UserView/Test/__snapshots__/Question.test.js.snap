// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserTestQuestion /> render() renders the component 1`] = `
<div
  className="content"
>
  <Card
    width="800px"
  >
    <WithStyles(Typography)
      component="h2"
      variant="headline"
    >
      Assignment Total - 
    </WithStyles(Typography)>
    <WithStyles(Typography)
      component="p"
    >
      All answers must be your own
    </WithStyles(Typography)>
    <br />
    <div
      className="row alignRight"
    >
      <WithTheme(ButtonWithTheme)
        color="secondary"
        onClick={[Function]}
      >
        Begin Assignment
      </WithTheme(ButtonWithTheme)>
    </div>
  </Card>
</div>
`;
